chapters/10_chapter_9.tex

```diff
@@ -0,0 +1,329 @@
+\chapter{Fault Attacks} \label{c9_ninthchapter:cha}
+
+% Swap the definition of \abs* and \norm*, so that \abs
+% and \norm resizes the size of the brackets, and the 
+% starred version does not.
+\makeatletter
+\let\oldabs\abs
+\def\abs{\@ifstar{\oldabs}{\oldabs*}}
+%
+\let\oldnorm\norm
+\def\norm{\@ifstar{\oldnorm}{\oldnorm*}}
+\makeatother
+
+\title{
+    {\large 372-2-5421} \\
+    {\large \textsc{Attacks on Implementations of Secure Systems}} \\
+    \vspace{20pt}
+    \textbf{\textsc{Scribe notes} -- Fault Attacks}
+}
+\author{
+    \parbox{\linewidth}{\centering
+    Dorel Yaffe, Dan Arad, Iliya Fayans \\
+    \vspace{7pt}\texttt{\small{\{yaffed,arda,fayansi\}@post.bgu.ac.il}}
+    }
+}
+\date{\footnotesize{\textbf{2019-05-27}}}
+
+\maketitle
+
```

Comment:
This is already done in the main. If you want to add your name, add it in the main. Don't touch the title or date.

Reply:
Removed